src/components/action-popover/action-popover.pw.tsx

```diff
@@ -820,6 +820,14 @@ test.describe("check props for ActionPopover component", () => {
       await expect(itemChevron).toHaveAttribute("type", chevronType);
     });
   });
+
+  test("should render with aria-label prop", async ({ mount, page }) => {
```

Comment:
**comment**: Happy to have this left in, however I feel like we can just cover this with a unit test